src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java

```diff
@@ -300,7 +300,114 @@ public static boolean isOfType(int type, int... types) {
      * @return true if type matches one of the given types.
      */
     public static boolean isOfType(DetailAST ast, int... types) {
-        return ast != null && isOfType(ast.getType(), types);
+        return isOfType(ast.getType(), types);
+    }
+
+    /**
+     * Determines if the token type belongs to the given first parameter.
+     *
+     * @param type the Token Type to check
+     * @param first the acceptable token
+     *
+     * @return true if type equals to first
+     */
+    public static boolean isOfType(int type, int first) {
+        return type == first;
+    }
+
+    /**
+     * Determines if the AST belongs to the given first parameter.
+     *
+     * @param ast the AST node to check
+     * @param first the acceptable token
+     *
+     * @return true if type matches one of the given types.
+     */
+    public static boolean isOfType(DetailAST ast, int first) {
+        return isOfType(ast.getType(), first);
+    }
```

Comment:
Please don't remove null check in this method. This condition is already covered and used, see https://github.com/checkstyle/checkstyle/pull/11543#issuecomment-1101406256. So UTs and the report generation will fail on this line.

Reply:
I mistakenly did that. Doing the changes.